prow/plugins/config.go

```diff
@@ -982,3 +983,135 @@ func (pluginConfig *ProjectConfig) GetOrgColumnMap(org string) map[string]string
 	}
 	return nil
 }
+
+// Bugzilla holds options for checking Bugzilla bugs in a defaulting hierarchy.
+type Bugzilla struct {
+	// Default settings mapped by branch in any repo in any org.
+	// The `*` wildcard will apply to all branches.
+	Default map[string]BugzillaBranchOptions `json:"default"`
```

Comment:
The implementation looks solid to me, just wondering how common use cases that benefit from this are since it adds complexity.

Reply:
Yeah, since the field validation on the Bugzilla bugs is going to be on the target release, you'll want to make sure all `release-1.13` branch PRs target `1.13` version or similar, across an org. If we had a config like that for `branch-protector` you couldn't do that